www/source/docs/get-habitat.html.md

```diff
@@ -23,7 +23,14 @@ _Habitat for Linux requires a 64-bit processor with a kernel greater than 2.6.32
 <a class="button" href="https://api.bintray.com/content/habitat/stable/linux/x86_64/hab-%24latest-x86_64-linux.tar.gz?bt_package=hab-x86_64-linux">Download Habitat for Linux</a>
 
 ### For Windows
-Habitat currently runs on Mac and Linux operating systems only. Windows support is coming soon.
+To start using Habitat on Windows, download the following binary.  
+If you intend to build Habitat packages on your Windows PC, you will also need to install [Docker for Windows](https://docs.docker.com/docker-for-windows/).  
+_Habitat for Windows requires 64bit Windows 10 Pro, Enterprise and Education (1511 November update, Build 10586 or later) and Microsoft Hyper-V.
```

Comment:
We may want to validate the ansi console sequences work right on 1511.  i've been testing on the anniversary update 1607.  cc @davidwrede